modules/nf-core/gemma/kinshipmatrix/main.nf

```diff
@@ -0,0 +1,49 @@
+process GEMMA_KINSHIPMATRIX {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gemma:0.98.5--ha36d3ea_0':
+        'biocontainers/gemma/0.98.5--ha36d3ea_0' }"
+
+    input:
+    tuple val(meta), path(genotype)
+    tuple val(meta2), path(phenotype)
```

Comment:
As both files represent different information from the same dataset, they should be kept in sync and therefore in the same tuple.
```suggestion
    tuple val(meta), path(genotype), path(phenotype)
```